app/uk/gov/hmrc/eoricommoncomponent/frontend/views/start.scala.html

```diff
@@ -64,7 +64,7 @@ <h2 class="govuk-heading-m" id="gb-eori-heading">@messages("ecc.start-page.gbeor
         </ul>
         <p class="govuk-body" id="gb-eori-para2">@Html(messages("ecc.start-page.gbeori.para2"))</p>
 
-        <h2 class="govuk-heading-m" id="gb-eori-heading">@messages("ecc.start-page.solo.title")</h2>
+        <h2 class="govuk-heading-m" id="gb-eori-heading-solo">@messages("ecc.start-page.solo.title")</h2>
```

Comment:
Not going to block the ticket but would be better as "sole-trader" instead of "solo"

Reply:
Done